app/src/main/java/com/example/petmanagerdemomvp/activity/MainActivity.kt

```diff
@@ -0,0 +1,144 @@
+package com.example.petmanagerdemomvp.activity
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AlertDialog
+import com.example.petmanagerdemomvp.Contract
+import com.example.petmanagerdemomvp.PetAdapter
+import com.example.petmanagerdemomvp.R
+import com.example.petmanagerdemomvp.model.Pet
+import com.example.petmanagerdemomvp.model.Repository
+import com.example.petmanagerdemomvp.presenter.PetPresenter
+import com.example.petmanagerdemomvp.util.Checkout
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.edit_item.view.*
+
+class MainActivity : AppCompatActivity(), Contract.View, View.OnClickListener{
+    private lateinit var petPresenter: PetPresenter
+    private lateinit var petAdapter: PetAdapter
+    private lateinit var alertDialog: AlertDialog
```

Comment:
k khuyến khích dùng lateinit var nha e
```suggestion
private var petPresenter: PetPresenter? = null
private var petAdapter: PetAdapter? = null
private var alertDialog: AlertDialog? = null
```